CHANGELOG.md

```diff
@@ -337,7 +337,7 @@ client libraries.
 ### Supported functionality
 
 - Create a REST or Realtime instance by passing `ClientOptions`:
-    - `ClientOptions` can be created by passing an API token (`ClientOptions.fromKey`)
+    - `ClientOptions` can be created by passing an API token (`ClientOptions(key: key)`)
```

Comment:
We shouldn't be modifying old change log entries, unless they were wrong at the point of release they were made for (in this case, [release tag `1.0.0+dev.1`](https://github.com/ably/ably-flutter/releases/tag/1.0.0%2Bdev.1)).

Reply:
Sorry, I ran search & replace across all MD files and I missed that this one was in CHANGELOG 🤦 Reverted this in https://github.com/ably/ably-flutter/pull/348/commits/155469282583605d8d9340ca22adbc133bc90b48